include/libdnf/module/module_item.hpp

```diff
@@ -138,13 +140,19 @@ class ModuleItem {
     // @replaces libdnf:module/ModuleItem.hpp:method:ModuleItem.getStaticContext()
     bool get_static_context() const;
 
-    /// @return The list of strings of modules requires (e.g. "nodejs", "nodejs:12", "nodejs:-11").
-    /// @param remove_platform When true, the method will not return requires with stream "platform"
-    //                         (default is false).
+    /// @return Module dependencies.
     /// @since 5.0
     //
     // @replaces libdnf:module/ModuleItem.hpp:method:ModuleItem.getRequires(bool removePlatform=false)
-    std::vector<std::string> get_requires(bool remove_platform = false);
+    std::vector<ModuleDependency> get_module_dependencies() const;
+
+    /// @return The string representing module dependencies (e.g. "ninja;platform:[f29]" or "nodejs:[-11]").
+    /// @param remove_platform When true, the method will not return dependencies with stream "platform"
+    ///                        (default is false).
+    /// @since 5.0
+    //
+    // @replaces libdnf:module/ModuleItem.hpp:method:ModuleItem.getRequires(bool removePlatform=false)
+    std::string get_module_dependencies_string(bool remove_platform = false);
```

Comment:
What about to move it to private section? I guess it is too specific to keep it in public API.

Reply:
I don't know how to test it if I move it to private. But I can remove it from the test for now, until there is another way.